ui/start.html

```diff
@@ -4,7 +4,8 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="flag-icon-css/css/flag-icon.css">
     <link rel="stylesheet" href="css/fonts.css">
-    <link rel="stylesheet" href="css/main.css">
+    <link rel="stylesheet" href="css/main.css" media="screen and (max-height: 1080px)"> 
+    <link rel="stylesheet" href="css/main-1080.css" media="screen and (min-height: 1080px)">
```

Comment:
Which rule is chose if the height is exactly 1080?

Reply:
Oh, it would load them both, but the 1080 one would have a higher priority. Should I reduce some pixels off of max-height?